packages/common/validators/src/validators/is-birth-date-6.ts

```diff
@@ -1,5 +1,27 @@
 /** @tossdocs-ignore */
 export function isBirthDate6(birthDate: string) {
-  const re = /^[0-9]{2}(?:0[1-9]|1[0-2])(?:0[1-9]|[1,2][0-9]|3[0,1])$/;
-  return re.test(birthDate);
+  if (!/^\d{6}$/.test(birthDate)) {
+    return false;
+  }
+
+  const year = parseInt(birthDate.substring(0, 2), 10);
+  const month = parseInt(birthDate.substring(2, 4), 10);
+  const day = parseInt(birthDate.substring(4, 6), 10);
+
+  if (month < 1 || month > 12) {
+    return false;
+  }
```

Comment:
A small suggestion, but why not define the numbers 1 and 12 as `JANUARY` and `DECEMBER` constants to manage them? 🤗
```suggestion
  if (month < JANUARY || month > DECEMBER) {
    return false;
  }
```

Reply:
I like This comment awesome..!

Reply:
I love it too.
This code prevents the use of magic numbers

Reply:
@ssi02014 Thanks! I agree with you.[refactor: change from magic number to constant variable](https://github.com/toss/slash/pull/420/commits/42f7101396dcf0e6e238c97652bd340b37a57f6f) (cc. @minsoo-web, @okinawaa )